credentials/google/google.go

```diff
@@ -64,14 +64,25 @@ func NewDefaultCredentials() credentials.Bundle {
 //
 // This API is experimental.
 func NewComputeEngineCredentials() credentials.Bundle {
+	return NewComputeEngineCredsWithPerRPC(nil)
+}
+
+// NewComputeEngineCredsWithPerRPC returns a credentials bundle that is configured to work
+// with google services. This API must only be used when running on GCE.
+//
+// This API is experimental.
+func NewComputeEngineCredsWithPerRPC(perRPC credentials.PerRPCCredentials) credentials.Bundle {
```

Comment:
Let make this `NewComputeEngineCredsWithOptions`, and take a struct as the input. We may need this in the future to support other features (e.g. mtls)

Reply:
Done